garage/tf/exploration_strategies/ou_strategy.py

```diff
@@ -81,7 +81,8 @@ def get_action(self, t, observation, policy, **kwargs):
         return np.clip(action + ou_state, self.action_space.low,
                        self.action_space.high), agent_infos
 
-    def get_actions(self, observations, policy):
+    @overrides
```

Comment:
this file also doesn't seem to have any tf dependencies. move out of the TF tree?

Reply:
yes

Reply:
I checked the original `garage.exploration_strategies.ou_strategy` is not used anywhere. I will remove that and move the tf one there.

Reply:
sgtm